packages/webapp/src/components/Animals/AddAnimalsFormCard/AnimalSelect.tsx

```diff
@@ -96,7 +96,7 @@ export function AnimalBreedSelect<T extends FieldValues>({
           placeholder={isTypeSelected ? undefined : t('ADD_ANIMAL.BREED_PLACEHOLDER_DISABLED')}
           isDisabled={!isTypeSelected || isDisabled}
           onChange={(option) => onChange(option)}
-          value={value}
+          value={value || null}
```

Comment:
`undefined` doesn't clear the select, but `null` does.

https://github.com/user-attachments/assets/6d3d6086-49bb-4eb3-a5d5-1fcae0f43a11